analyzers/tests/SonarAnalyzer.UnitTest/Helpers/DebugHelperTest.cs

```diff
@@ -44,8 +43,8 @@ public class DebugHelperTest
         private static void SetupRunAndReset(string value, bool shouldBeTrue)
         {
             // Arrange
-            var oldValue = Environment.GetEnvironmentVariable(DebugHelper.AnalyzerInternalDebugEnvVariable);
-            Environment.SetEnvironmentVariable(DebugHelper.AnalyzerInternalDebugEnvVariable, value);
+            using var scope = new EnvironmentVariableScope();
+            scope.SetVariable(DebugHelper.AnalyzerInternalDebugEnvVariable, value);
```

Comment:
Step 1: This could be really useful as a get/set property on the scope.
Step 2: Step back, investigate
Step 3: We can just delete this entire file altogether with `DebutgHelper.cs` It's never used.

Reply:
There is no need to take step 1 and 2 if you have the third one :) #lessismore